httpcore5/src/main/java/org/apache/hc/core5/http/protocol/RequestContent.java

```diff
@@ -110,21 +112,56 @@ public void process(final HttpRequest request, final EntityDetails entity, final
             }
         }
         if (entity != null) {
+
```

Comment:
@arturobernalg Nooooooooo. Cmon! Some entities stream their content and their length is not known it advance! This is what you have `EntityDetails` for.

Reply:
@ok2c  Corrected.